modules/orchestrator/initialize.go

```diff
@@ -251,6 +251,8 @@ func initLeaderCron(ep *endpoints.Endpoints, ctx context.Context) error {
 
 	go loop.New(loop.WithContext(ctx), loop.WithInterval(5*time.Minute)).Do(ep.SyncAddonResources)
 
+	go loop.New(loop.WithContext(ctx), loop.WithInterval(10*time.Minute)).Do(ep.CleanRemainingAddonAttachment)
```

Comment:
10 minutes is too frequent, I suggest 1 hours

Reply:
done